mne/preprocessing/ieeg/_projection.py

```diff
@@ -106,3 +107,92 @@ def project_sensors_onto_brain(info, trans, subject, subjects_dir=None,
         montage_kwargs['ch_pos'][info.ch_names[idx]] = loc
     info.set_montage(make_dig_montage(**montage_kwargs))
     return info
+
+
+@verbose
+def project_sensors_onto_inflated(info, trans, subject, subjects_dir=None,
```

Comment:
Keep private as per suggestion in https://github.com/mne-tools/mne-python/pull/11437#issuecomment-1413949398 (i.e., purely used internally for fixing/adjusting visualization, users should never see or know about this private function)
```suggestion
def _project_sensors_onto_inflated(info, trans, subject, subjects_dir=None,
```